javascript-forms/form-values.js

```diff
@@ -0,0 +1,12 @@
+var $input = document.querySelector('#contact-form');
+
+function onSubmit(event) {
+  event.preventDefault();
+  var values = {};
+  values.name = $input[0].value;
+  values.email = $input[1].value;
+  values.message = $input[2].value;
```

Comment:
Rather than using indexes on the form, you should make use of the named properties in the `elements` property on the `form` DOM element object